### DISCUSSION TITLE
8 and 24 bpp support

### DISCUSSION THREAD

**Discussion by Sanjubisanal** [FreeRDP/FreeRDP#7409] [Q&A]
Does freerdp supports 8 and 24 bit bpp.

**Comment by akallabeth** [comment]
yes.

**Reply by eronnen** [reply]
Is there a way to force 8 bpp? I noticed that even when I specify 8 bpp in the command line, it uses 16 bpp:

https://github.com/FreeRDP/FreeRDP/blob/master/libfreerdp/core/capabilities.c#L349

**Reply by akallabeth** [reply]
Only the server can do that.
FYI: I´d recommend using `/network:auto` over any custom options as the `/bpp:xx` setting only affects bitmap modes and not modern `RFX` or `H264` modes. (that is, unless you connect to windows xp)

**Reply by Sanjubisanal** [reply]
https://github.com/FreeRDP/FreeRDP/blob/41d03143b85d0c531e44b402d6979fe445dcdf19/server/shadow/shadow_client.c#L411

But shadow doesn't work with 24 bit in actual
